examples/android-room/src/main/java/io/sentry/android/roomsample/ui/MainActivity.kt

```diff
@@ -119,6 +122,16 @@ class TrackAdapter : RecyclerView.Adapter<TrackAdapter.ViewHolder>() {
                 ).putExtra(EditActivity.TRACK_EXTRA_KEY, track)
             )
         }
+        holder.row.infoButton.setOnClickListener {
+            val context = holder.row.context
```

Comment:
maybe we separate the `TrackAdapter` into a new class, to keep a bit more organized.